fdmt/run.py

```diff
@@ -0,0 +1,116 @@
+if __name__ == '__main__':
+    import argparse
+    from .cpu_fdmt import FDMT
+    import numpy as np
+    
+    try:
+        import blimpy
```

Comment:
maybe add `blimpy` to the requirements in `setup.cfg`?

Reply:
aha, I didn't know this was a thing. I have added it now